ext/standard/tests/mail/gh7875.phpt

```diff
@@ -3,6 +3,11 @@ GH-7875 (mails are sent even if failure to log throws exception)
 --INI--
 sendmail_path={MAIL:{PWD}/gh7875.mail.out}
 mail.log={PWD}/gh7875.mail.log
+--SKIPIF--
+<?php
+if (!extension_loaded('posix')) die('skip POSIX extension not loaded');
+if (posix_geteuid() == 0) die('skip Cannot run test as root.');
+?>
```

Comment:
This skips the test on Windows, although it otherwise would work fine. On Windows, there is no need to check the euid at all, which could easily be fixed. However, the follow-up fix (d2c663441d0964fe8700b25a2d0bbeb313390a0d) which is fine per se, makes it more difficult to also run the test on Windows (due to having `posix` in the `--EXTENSION--` section). Maybe we should split this test. Or, if we expect more such `posix_geteuid() == 0` (should probably better be `posix_geteuid() === 0`) checks, then it may make sense to come up with a more general solution (possibly a new marker section which does this check and the skipping behind the scenes). @iluuu1994, thoughts?

Reply:
I didn't think about Windows. TBH I didn't look into why exactly the test was failing with `root`, but for this test it's probably fine to remove `posix` from `EXTENSIONS` and skip the `root` check for Windows.